traffic_ops/v4-client/deliveryservice.go

```diff
@@ -362,6 +362,22 @@ func (to *Session) GetDeliveryServiceURISigningKeys(dsName string, header http.H
 	return []byte(data), reqInf, nil
 }
 
+// CreateDeliveryServiceURISigKeys creates new URI-signing keys used by the Delivery Service
+// identified by the XMLID 'dsName'
+func (to *Session) CreateDeliveryServiceURISigKeys(dsName string, header http.Header, body map[string]tc.URISignerKeyset) (tc.Alerts, toclientlib.ReqInf, error) {
+	var alerts tc.Alerts
+	reqInf, err := to.post(fmt.Sprintf(APIDeliveryServicesURISigningKeys, dsName), body, header, &alerts)
+	return alerts, reqInf, err
+}
+
+// DeleteDeliveryServiceURISigKeys deletes the URI-signing keys used by the Delivery Service
+// identified by the XMLID 'dsName'
+func (to *Session) DeleteDeliveryServiceURISigKeys(dsName string, header http.Header) (tc.Alerts, toclientlib.ReqInf, error) {
```

Comment:
Should we change the `dsName` param to `dsXMLId` or something like that, to avoid confusion?